content/applications/general/users/2fa.rst

```diff
@@ -1,127 +1,123 @@
-when you try to log in.
+Two-factor authentication (2FA) is a good way to improve security, and to prevent unauthorized
+persons from accessing user accounts.
 
-This means an attacker needs *both* to have guessed (or found) your
-password and to access (or steal) your authenticator, a more difficult
-proposition than either one or the other.
+Practically, |2fa| means storing a secret inside an *authenticator*, usually on a mobile phone, and
+exchanging a code from the authenticator when trying to log in.
+
+This means an attacker needs to guess the account password *and* have access to the authenticator,
+a more difficult proposition than either one or the other.
 
 Requirements
 ============
 
-.. note:: These lists are just examples, they are not endorsements of
-          any specific software.
+.. important::
+   These lists are just examples, they are not endorsements of any specific software.
+
+Phone-based authenticators are the easiest and most commonly used. Examples include:
 
-If you don't already have one, you will need to choose an
-authenticator.
+- `Authy <https://authy.com/>`_
+- `FreeOTP <https://freeotp.github.io/>`_
+- `Google Authenticator <https://support.google.com/accounts/answer/1066447?hl=en>`_
+- `LastPass Authenticator <https://lastpass.com/auth/>`_
+- `Microsoft Authenticator
+  <https://www.microsoft.com/en-gb/account/authenticator?cmp=h66ftb_42hbak>`_
 
-Phone-based authenticators are the easiest and most common so we will
-assume you'll pick and install one on your phone, examples include
-`Authy <https://authy.com/>`_, `FreeOTP
-<https://freeotp.github.io/>`_, `Google Authenticator
-<https://support.google.com/accounts/answer/1066447?hl=en>`_,
-`LastPass Authenticator <https://lastpass.com/auth/>`_, `Microsoft
-Authenticator
-<https://www.microsoft.com/en-gb/account/authenticator?cmp=h66ftb_42hbak>`_,
-...; password managers also commonly include :abbr:`2FA (two-factor
-authentication)` support e.g. `1Password
-<https://support.1password.com/one-time-passwords/>`_, `Bitwarden
-<https://bitwarden.com/help/article/authenticator-keys/>`_, ...
+Password managers are another option. Common examples include:
+- `1Password <https://support.1password.com/one-time-passwords/>`_
+- `Bitwarden <https://bitwarden.com/help/article/authenticator-keys/>`_,
 
-For the sake of demonstration we will be using Google Authenticator
-(not because it is any good but because it is quite common).
+.. note::
+   The remainder of this document uses Google Authenticator as an example, as it is one of the most
+   commonly used. This is not an endorsement of the product.
 
 Setting up two-factor authentication
 ====================================
 
-Once you have your authenticator of choice, go to the Odoo instance
-you want to setup :abbr:`2FA (two-factor authentication)`, then open
-:guilabel:`Preferences` (or :guilabel:`My Profile`):
+After selecting an authenticator, log in to Odoo, then navigate to :menuselection:`My Profile`.
```

Comment:
```suggestion
After selecting an authenticator, log in to Odoo, then click the profile avatar in the upper-right corner, and select :guilabel:`My Profile` from the resulting drop-down menu.
```
SUGGESTION: just to add some more specific directions about how to access the My Profile page... 
a supplemental screenshot of this could also be beneficial, but it's totally your call 👍